BridgeAppSDKTests/MockObjects/MockBridgeInfo.h

```diff
@@ -56,5 +56,6 @@
 @property (nonatomic, readonly, copy) NSString * _Nullable keychainService;
 @property (nonatomic, readonly, copy) NSString * _Nullable keychainAccessGroup;
 @property (nonatomic, readonly, copy) NSString * _Nullable appGroupIdentifier;
+@property (nonatomic, readonly, copy) NSString * _Nullable privacyPolicyLinkURL;
```

Comment:
should this be privacyPolicyLinkURLString? Or does it not actually need to match the SBABridgeInfo protocol?